### DISCUSSION TITLE
多次执行脚本只输出”. %“

### DISCUSSION THREAD

**Discussion by KayaiJia** [NewFuture/DDNS#294] [Q&A] [answered]
配置好config之后第一次执行可以正常获取和更改地址，后续再次执行就只输出一个“. %“了。不知道这是bug还是就这样设计的？
![未命名1](https://user-images.githubusercontent.com/37146022/145548502-c8c0f40d-007c-4789-a51c-bb78d1953c24.jpg)
![未命名](https://user-images.githubusercontent.com/37146022/145548527-273cdcc9-b2c2-4f2b-852a-1d1d14593f7f.jpg)

**Comment by NewFuture** [comment, marked as answer]
ip没有任何变化时(跳过更新)输出 `.`

**Reply by KayaiJia** [reply]
感谢！
